src/_locales/zh_CN/messages.yml

```diff
@@ -418,6 +418,9 @@ labelNotifyUpdates:
 labelNotifyUpdatesGlobal:
   description: Option to prioritize global notification option over script's setting.
   message: 忽略每个脚本的提示（编辑器中的“设置”页）
+labelUpdatesParallel:
+  description: 当多个脚本的更新地址为同一个域时，会根据设置的并发数进行分批次更新
```

Comment:
Please run `yarn i18n` to normalize the i18n files before pushing

Reply:
FYI the `description` is for the translator (who understands English) to understand what this key is for, so it doesn't need to be translated. Only `message` should be translated.

Reply:
understand, thank you for your description